frontend/components/panels/AllowanceAmountCheckPanel.vue

```diff
@@ -1,24 +1,30 @@
 <template>
     <BasePanel :current-state="currentStateAndTitle.name" class="WalletDaiDepositCheckPanel">
         <template #title>{{ currentStateAndTitle.title }}</template>
-        <TextBlock v-if="isExplanationsShown">
-            To bid on an auction with DAI, first funds need to be deposited to the
-            <Explain text="VAT">
-                The
-                <a
-                    href="https://docs.makerdao.com/smart-contract-modules/core-module/vat-detailed-documentation#2-contract-details"
-                    >VAT contract</a
-                >
-                is the core vault engine of the Maker Protocol and manages the central accounting invariants of DAI.
-                Depositing and interacting with the VAT is necessary in order to participate in auctions. </Explain
-            >. The following transaction authorizes the wallet address to deposit into the VAT. It is a prerequisite to
-            participate in the auction.
-        </TextBlock>
+        <div v-if="isExplanationsShown">
+            <TextBlock v-if="currency === 'DAI'">
+                To bid on an auction with DAI, first funds need to be deposited to the
+                <Explain text="VAT">
+                    The
+                    <a
+                        href="https://docs.makerdao.com/smart-contract-modules/core-module/vat-detailed-documentation#2-contract-details"
+                        >VAT contract</a
+                    >
+                    is the core vault engine of the Maker Protocol and manages the central accounting invariants of
+                    DAI. Depositing and interacting with the VAT is necessary in order to participate in auctions. </Explain
+                >. The following transaction authorizes the wallet address to deposit into the VAT. It is a
+                prerequisite to participate in the auction.
+            </TextBlock>
+            <TextBlock v-if="currency === 'MKR'">
+                In order to move funds between wallet and Flap the Flap contract address needs to be authorized. Hence
```

Comment:
this sentence reads funny.

Reply:
Ill hand this over to @LukSteib!

Reply:
Agree. Let's change to: 
"In order to move funds the Flap contract address needs to be authorized. ..."